agent/resources/cgroup/factory/mock_factory/mock_cgroup_factory_linux.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`agent/resources` package is going away with PR https://github.com/aws/amazon-ecs-agent/pull/1382/. These changes can be excluded from this PR.

Reply:
i figured since these were generated, we can get rid of everything together during task-resources merge time.